packages/internet-header/src/components/post-main-navigation/post-main-navigation.tsx

```diff
@@ -250,6 +250,7 @@ export class PostMainNavigation implements HasDropdown, IsFocusable {
         <nav
           id="post-internet-header-main-navigation"
           class={{ 'main-navigation': true, 'open': this.mobileMenuOpen }}
+          role="menu"
```

Comment:
Do you think the main navigation should have the role menu also on desktop or only on mobile devices?

Reply:
From what I understood of the [ARIA: menu role documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role) it seems to be appropriate for the desktop version as well. 

> A menu can be a permanently visible list of controls or a widget that can be made to open and close.